pkg/addons/registry/registry.go

```diff
@@ -181,6 +174,29 @@ func (o *Registry) Outro(ctx context.Context, cli client.Client) error {
 		return fmt.Errorf("unable to create registry-auth secret: %w", err)
 	}
 
+	// create a secret to allow accessing the seaweedfs s3
+	rwKey, rwSecret := seaweedfs.GetRWInfo()
+	s3AccessSecret := corev1.Secret{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Secret",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "seaweedfs-s3-rw",
+			Namespace: namespace,
+		},
+		StringData: map[string]string{
+			"s3AccessKey": rwKey,
+			"s3SecretKey": rwSecret,
+		},
+		Type: "Opaque",
+	}
+	err = cli.Create(ctx, &s3AccessSecret)
```

Comment:
Ditto: would this be executed/fail on upgrades since the secret will already exist?